docs/data/base/pages.ts

```diff
@@ -1,9 +1,14 @@
+/// <reference path="../../types/icons.d.ts"/>
+import ArticleRoundedIcon from '@mui/icons-material/ArticleRounded';
+import ToggleOffRoundedIcon from '@mui/icons-material/ToggleOffRounded';
+import CodeRoundedIcon from '@mui/icons-material/CodeRounded';
+import VisibilityRoundedIcon from '@mui/icons-material/VisibilityRounded';
 import pagesApi from 'docs/data/base/pagesApi';
 
 const pages = [
   {
     pathname: '/base/getting-started',
-    icon: 'DescriptionIcon',
+    icon: ArticleRoundedIcon,
```

Comment:
I am afraid that this could lead to inconsistency if someone forgets to change the icon on other pages (Material UI, Joy UI, and Base UI follow the same structure). I think it makes more sense for the core pages to reuse the same icons.

Reply:
Could we export the icons from a doc file ?
Something like 

```tsx
import { DescriptionIcon } from '../some/endpoint/doc-pages-icons';
```

Reply:
I moved to a solution where `AppNavIcons.ts` exports the object of icons available. and pages use this object.

For core pages, it means the following diff

```diff
- icons: "DescriptionIcon",
+ icons: standardNavIcons.DescriptionIcon,
```

So you still have a centralized list of standard icons. It's simpler to modify thanks to TS autocomplete, and others can still have access to those icons